### DISCUSSION TITLE
Hire someone to build custom Step Mania

### DISCUSSION THREAD

**Discussion by peaktechnologies** [stepmania/stepmania#2246] [General]
Hi there, 
I'm interested in hiring someone that could build an instance of Step Mania for my company. We have a client of ours who would like a version of the game built with their company colors / branding GFX. 
I'm not sure how / who to reach out to, but if you have built a running Version of Step Mania for yourself and are interested in taking on a project to build another version please be sure to comment here and we can connect to discuss details. I was hoping to find a way to connect with the team who built Step Mania and inquire with them directly, however the forums on Step Mania's website seem to be shut down. 
Thank you

**Comment by Horsey-** [comment]
I’m happy to chat about it. Send me a DM

**Reply by peaktechnologies** [reply]
Thanks for your reply, I'm not sure how to DM here in Github, if that's a thing, but shoot me an email and we can chat some more. 
richard@peakt.com
I'll be happy to give you some more details about the job.
